src/openms/source/CHEMISTRY/ModificationsDB.cpp

```diff
@@ -344,26 +323,13 @@ namespace OpenMS
 
           for (vector<String>::iterator orig_it = origins.begin(); orig_it != origins.end(); ++orig_it)
           {
-            if (orig_it->size() == 1)
+            if ((orig_it->size() == 1) && (*orig_it != "B") && (*orig_it != "J") && (*orig_it != "Z"))
```

Comment:
I see, might be hard to track down the reason for others so maybe add a comment ?